mods/cnc/maps/funpark01/map.yaml

```diff
@@ -0,0 +1,544 @@
+Selectable: False
+
+MapFormat: 6
```

Comment:
This needs to become `MapFormat: 7` since https://github.com/openra/openra/pull/7134.